hypothesis-python/tests/cover/test_statistical_events.py

```diff
@@ -202,21 +200,6 @@ def test(n):
     assert "shrunk example" in stats.exit_reason
 
 
-@pytest.mark.parametrize("drawtime,runtime", [(1, 0), (-1, 0), (0, -1), (-1, -1)])
-def test_weird_drawtime_issues(drawtime, runtime):
-    # Regression test for #1346, where we don't have the expected relationship
-    # 0<=drawtime<= runtime due to changing clocks or floating-point issues.
-    engine = ConjectureRunner(lambda: None)
-    engine.exit_reason = ExitReason.finished
-    engine.status_runtimes[Status.VALID] = [0]
-
-    engine.all_drawtimes.append(drawtime)
-    engine.all_runtimes.extend([0, runtime])
-
-    stats = Statistics(engine)
-    assert stats.draw_time_percentage == "NaN"
```

Comment:
Briefly skimming #1346, this bug was present in Python 2 and 3 – why remove the test?

Reply:
Because it ought to be impossible on Python 3, and I didn't go back and read the issue :sob: 

Will revert.